pyproject.toml

```diff
@@ -20,6 +20,8 @@ fastapi = ">=0.97.0,<0.99.0"
 genson = "^1.2.2"
 click = "^8.1.3"
 coloredlogs = "^15.0.1"
+urllib3 = "^2.0.7"
+pydantic = "^1.10.13"
```

Comment:
I wouldn't want to add these as direct dependencies, rather just update them in the lock file.